graylog2-web-interface/src/views/components/widgets/ExtraWidgetActions.tsx

```diff
@@ -32,13 +32,15 @@ const ExtraWidgetActions = ({ onSelect, widget }: Props) => {
   const pluginWidgetActions = usePluginEntities('views.widgets.actions');
   const extraWidgetActions = useMemo(() => pluginWidgetActions
     .filter(({ isHidden = () => false }) => !isHidden(widget))
-    .map(({ title, action, type }) => {
+    .map(({ title, action, type, disabled }) => {
```

Comment:
We can also use a default value (`() => false`) for `disabled`. This would make the check in line 41 simpler and ensures that there is always a boolean value returned.